pkg/network/sdn-cni-plugin/openshift-sdn.go

```diff
@@ -83,8 +94,9 @@ func (p *cniPlugin) doCNI(url string, req *cniserver.CNIRequest) ([]byte, error)
 
 // Send the ADD command environment and config to the CNI server, returning
 // the IPAM result to the caller
-func (p *cniPlugin) CmdAdd(args *skel.CmdArgs) (types.Result, error) {
-	body, err := p.doCNI("http://dummy/", newCNIRequest(args))
+func (p *cniPlugin) doCNIServerAdd(req *cniserver.CNIRequest, hostVeth string) (types.Result, error) {
+	req.Env["OSDN_HOSTVETH"] = hostVeth
```

Comment:
Any reason to do this in the environment rather than just stuff it into the JSON by extending CNIRequest?